R/utils.R

```diff
@@ -398,36 +398,41 @@ remap_trans_arg <- function(args, user_env = rlang::caller_env(2)) {
 
 ##' Rotate axis text
 ##'
-##' @param angle passed to [ggplot2::element_text]
-##' @param hjust passed to [ggplot2::element_text]
-##' @param vjust passed to [ggplot2::element_text]
+##' @param angle passed to [ggplot2::element_text()].
+##' @param hjust passed to [ggplot2::element_text()].
+##' @param vjust passed to [ggplot2::element_text()].
 ##' @param vertical if `TRUE`, then x-axis tick labels are rotated 90 degrees
-##' with `vjust` set to 0.5 and `hjust` set to 0; when `vertical` is set to
-##' `TRUE`, then `hjust` can be passed as character string that must match
-##' either `top` (then `hjust` is set to 1) or `bottom` (then `hjust` is set to
-##' 0
-##' @param ... passed to [ggplot2::element_text]
+##' with `vjust` set to 0.5 and `hjust` set to 1; when using `rot_y()`,
+##' y-axis tick labels are rotated 90 degrees with `hjust` is set to 0.5
```

Comment:
"with \`hjust\` is set to" -> "with \`hjust\` set to"?

Reply:
Thanks; fixed.